rdr_service/rdr_client/pairing_assigner.py

```diff
@@ -57,21 +83,44 @@ def main(client):
                 )
                 num_errors += 1
                 continue
+
             if not participant_id.startswith("P"):
                 logging.error(
-                    "Malformed participant ID from line %d: %r does not start with P.", reader.line_num, participant_id
+                    "Malformed participant ID from line %d: %r does not start with P.",
+                    reader.line_num,
+                    participant_id
                 )
                 num_errors += 1
                 continue
 
+            if biospecimen_sync:
+                request_url = "Participant/%s/BiobankOrder" % participant_id
+            elif pm_sync:
```

Comment:
please allow for all fields to be populated if both optional flags are added, not one choice or the other.